src/kademlia/lookup.ts

```diff
@@ -77,6 +82,7 @@ export class Lookup extends (EventEmitter as { new (): LookupEventEmitter }) {
     if (this.state === LookupState.Finished) {
       return;
     }
+    clearTimeout(this.timeout);
```

Comment:
strictNullChecks police over here. Should be `if (this.timeout) then`

Reply:
We have strict null checks turned on, this isn't an issue.

clearTimeout can be passed undefined / invalid timerId w/o issue, part of the spec.